pkg/device/sync.go

```diff
@@ -76,6 +76,7 @@ func probeDeviceMap() (map[string][]device, error) {
 
 func syncDrive(drive *types.Drive, device device) (updated bool) {
 	if string(drive.GetDriveName()) != device.Name {
+		// This can happen when the drive order changes on node reboots.
```

Comment:
It is possible to change drive name at runtime as well. You could remove this comment i.e. revert changes in this file.